addon/components/ember-table.js

```diff
@@ -191,13 +197,15 @@ StyleBindingsMixin, ResizeHandlerMixin, {
   }).property('_resolvedContent.[]'),
 
   // An array of Ember.Table.Row
-  footerContent: Ember.computed(function(key, value) {
-    if (value) {
-      return value;
-    } else {
-      return Ember.A();
+  footerContent: Ember.computed({
+    set: function(key, value) {
```

Comment:
to mimic the old code, this looks like it should be:
```
set: function(key, value) {
  return value;
},
get: function() {
  return Ember.A();
}
```
But I'm not actually sure what the old code is doing here.